garage/tf/samplers/off_policy_vectorized_sampler.py

```diff
@@ -151,12 +192,13 @@ def process_samples(self, itr, paths):
         """
         success_history = []
         for path in paths:
-            if "is_success" in path["env_infos"]:
-                success = np.array(path["env_infos"]["is_success"])
-                success_rate = np.mean(success)
-                success_history.append(success_rate)
+            success_history.append(
+                path["success_count"] / path["running_length"])
 
-        undiscounted_returns = [sum(path["rewards"]) for path in paths]
+        success_history = [
```

Comment:
why remove the checking `if 'is_success' in path['env_infos']`?

Reply:
`success_history` is used only for monitoring. Previously, if `is_success` is not reported in `env_infos`, the `sucess_rate` being logged will be zero. It is the same result effect as omitting `is_success` here.